policies/test_data.rego

```diff
@@ -0,0 +1,2108 @@
+package test_data
+
+# Generated automatically with `make data-to-rego`
+data := {
```

Comment:
It's pretty large - I guess we could prune out parts we know we don't care about.

Reply:
It would be nice to have a PipelineRun attestation here, and maybe we don't need the ConfigMap nor the PipelineRun objects?

Would it be possible to write up how to recreate this test data? I like the idea of embedding it here but this will eventually get stale and we'll struggle to figure out how to refresh it. Wdyt?

Reply:
To generate it you just do this `make data-to-rego`. It depends on code in the build-definitions repo, which made sense to me at the time.